Target.java

```diff
@@ -10,8 +10,27 @@
 import acm.program.*;
 import java.awt.*;
 
+
 public class Target extends GraphicsProgram {	
 	public void run() {
-		/* You fill this in. */
+		
+		double radius_1 = 72;
```

Comment:
The radius should have been declared as constants outside of the run method, also you should have named them outer_radius, middle_radius and inner_radius, that way someone knows which radius corresponds to which circl. So something like
 
```
private static final double OUTER_RADIUS= 72;
private static final double MIDDLE_RADIUS = OUTER_RADIUS * 0.65;
private static final double INNER_RADIUS = OUTER_RADIUS * 0.3;
```